docs/code/MlNetCookBook.md

```diff
@@ -578,6 +578,48 @@ var biases = modelParameters.GetBiases();
 
 ```
 
+## How do I look at the global feature importance?
+The below snippet shows how to get a glimpse of the the feature importance, or how much each column of data impacts the performance of the model.
+
+```csharp
+var transformedData = model.Transform(data);
+
+var featureImportance = context.Regression.PermutationFeatureImportance(model.LastTransformer, transformedData);
+
+foreach (var metricsStatistics in featureImportance)
+{
+    Console.WriteLine($"Root Mean Squared - {metricsStatistics.Rms.Mean}");
+}
+```
+
+## How do I get a model's weights to look at the global feature importance?
+The below snippet shows how to get a model's weights to help determine the feature importance of the model.
```

Comment:
>The below [](start = 0, length = 9)

Note that for a linear model, the weights are only an approximation. It helps to standardize the variables before the fit, so that they are all on the same scale, and even then, the linear regression solution does not account for correlations between the variables, and therefore this isn't a great measure of explainability.